### PR TITLE
Use RepoTags & RepoDigest in inspect

### LINKED ISSUE #17246: Tags in docker inspect contains digests
#13185 added `"Tags"` to `docker inspect` output but that array will also contain image digests and the output isn't really tags but `repo:tag` pairs(references). is this intentional? Maybe we should rename it.

`/images/json` uses keys `RepoTags` and `RepoDigests`

### PR DESCRIPTION
**Pull request by vdemeester**
To be coherent with /images/json (images command) 🐼

Closes #17246.
- [x] Needs docs/api update

/cc @tiborvass @tonistiigi 

🐸

Signed-off-by: Vincent Demeester vincent@sbr.pm

### CODE REVIEW

File: docs/reference/api/docker_remote_api_v1.21.md [hunk 1]
```diff
@@ -1550,7 +1550,10 @@ Return low-level information on the image `name`
           "Name" : "aufs",
           "Data" : null
        },
-       "Tags" : [
+       "RepoDigest" : [
```
Comment (tonistiigi) [inline docs/reference/api/docker_remote_api_v1.21.md]: RepoDigests

Comment (vdemeester) [inline docs/reference/api/docker_remote_api_v1.21.md]: Oh, typo typo :sweat_smile:

**Comment by tiborvass** [comment]
LGTM

**Comment by icecrime** [comment]
Please add this to the changelog if we intend to cherry-pick!

Otherwise LGTM.

**Comment by thaJeztah** [comment]
Just a minor nit: the original PR didn't mention that we return digests as well, so adding "and digests" to the API changelog would be nice to have; https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api.md#v121-api-changes 

```Markdown
`GET /images/(name)/json` now returns information about tags and digests of the image.
```

Otherwise LGTM

**Comment by tiborvass** [comment]
@vdemeester agree with @thaJeztah please update!

**Comment by vdemeester** [comment]
@tiborvass @thaJeztah ah true, updating :wink:

**Comment by thaJeztah** [comment]
:heart: thanks!

**Comment by tiborvass** [comment]
just for the record, CI was green right before new push :)

**Comment by vdemeester** [comment]
@tiborvass I believe you :heart_eyes_cat: :trollface: :stuck_out_tongue_closed_eyes:

**Comment by thaJeztah** [comment]
LGTM!

**Pull request merged**